test/runtests.jl

```diff
@@ -199,3 +201,23 @@ end
     @test_throws DimensionMismatch fit(KaplanMeier, [1, 2], [true])
     @test_throws ArgumentError fit(KaplanMeier, Float64[], Bool[])
 end
+
+@testset "Cox" begin
+    filepath = joinpath(Pkg.dir("Survival", "test"), "rossi.csv")
+    rossi = readtable(filepath)
+    rossi[:event] = EventTime.(rossi[:week],rossi[:arrest] .== 1)
+
+    outcome = coxph(@formula(event ~ 0+ fin+age+race+wexp+mar+paro+prio), rossi; tol = 1e-8)
+    outcome_coefmat = coeftable(outcome)
+
+    filepath_coefs = joinpath(Pkg.dir("Survival", "test"), "expected_coefmat.jld")
+    expected_coefmat = JLD.load(filepath_coefs, "expected_coefmat")
```

Comment:
Good point, also because I keep getting annoying warnings from `JLD`, it's probably just better to type in the values in the julia file.